analysis/q_gdn_ddcap/phase_d.py

```
@@ -0,0 +1,326 @@
+
+    nocap_df = load_exits_static(NO_CAP)
+    nocap_metrics = compute_metrics(nocap_df["static_pnl"].to_numpy())
+    nocap_metrics["path"] = str(NO_CAP.relative_to(REPO))
+
+    knife = evaluate_knife_edge(grid_results, anchor=ANCHOR)
+    mono_pf = evaluate_monotonicity(grid_results, "pf", lower_is_better=False)
+    mono_dd = evaluate_monotonicity(grid_results, "dd_pct", lower_is_better=True)
+
+    # ----- Console output -----
+    print("=" * 78)
+    print(f"Q-GDN-DDcap Phase D — sensitivity sweep at dailyDDCap ∈ {{2.2, 2.4, 2.6, 2.8, 3.0}}%")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)